src/Misc/layoutbin/installdependencies.sh

```diff
@@ -257,6 +256,28 @@ then
                 print_errormessage
                 exit 1
             fi
+        elif [ -e /etc/mariner-release ]
+        then
+            echo "The current OS is Mariner based"
+            echo "--------Mariner Version--------"
+            cat /etc/mariner-release
+            echo "------------------------------"
+
+            command -v yum
+            if [ $? -eq 0 ]
+                then
+                yum install -y icu
```

Comment:
Is this the only dependency we need to install on this OS?
As I can see on other OSs we install openssl/lttng-ust/...

Reply:
Yes, it is enough for Mariner OS. I have tested it and didn't mention any other dependencies that should be installed.